### ISSUE TITLE
It always crush and asks me to choose a home app

### ISSUE THREAD

**Issue by halawnss** [MM2-0/Kvaesitso#836]
I always mark " remember my choice " before choosing the launcher but it doesn't work for a long time before asking me the same thing again and again

![IMG_20240513_221930](https://github.com/MM2-0/Kvaesitso/assets/167705509/39946d23-8615-4ef8-9ecd-994a67897648)

**Comment by sig11b** [comment]
I have the same issue on an Android 10 device. There are no entries in the crash log. I attach a general debug log. The log covers several such crashes.
[kvaesitso-log-2024-05-19-094933.txt](https://github.com/MM2-0/Kvaesitso/files/15368854/kvaesitso-log-2024-05-19-094933.txt)

**Comment by sig11b** [comment]
A while after my previous comment, I switched to another launcher on that Android 10 device. However, with the release of 1.31.0 I gave kvaesitso another try on that Android 10 device. 
It's running for a few days now and there are no more crashes! :-) These crashes used to occur several times a day with 1.30. 
Something between 1.30 and 1.31 seemed to fix this issue. At least for me.

BTW #842 seems to be very similar.
